SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -91,6 +94,26 @@ class PaymentQueueController: NSObject, SKPaymentTransactionObserver {
         paymentQueue.remove(self)
     }
 
+	var unhandledTransactions: [Purchase] {
```

Comment:
This variable returns an array of `Purchase` objects whose transactions have not been removed from the queue yet.
I wonder if `unhandledTransactions` is the right name for this.